example/poisson/poisson_driver.cpp

```diff
@@ -59,13 +59,47 @@ TaskCollection PoissonDriver::MakeTaskCollection(BlockList_t &blocks) {
   const int num_partitions = pmesh->DefaultNumPartitions();
   TaskRegion &solver_region = tc.AddRegion(num_partitions);
 
+  // make some storage for doing an Allreduce
+  reductions.resize(1);
```

Comment:
Why is this a vector if it's always size 1? Why not make it a variable called `error` or something?

Reply:
There's no need for it to be a vector in this example.  It was one of those things I did to try to indicate that you could have multiple independent reductions going on simultaneously in a task region.

Reply:
This is helpful  for an example. We'll also be doing several independent reductions with potentially different operations.

Reply:
In that case can we reduce something else, like 1-phi^2 or some nonsense, just to show how that would work?